src/colmap/estimators/bundle_adjustment.cc

```diff
@@ -352,13 +352,13 @@ ceres::Solver::Options BundleAdjuster::SetUpSolverOptions(
       options_.max_num_images_direct_sparse_cpu_solver;
 
 #ifdef COLMAP_CUDA_ENABLED
```

Comment:
It would be nice to log a warning when the user selects `use_gpu=true` but either COLMAP or Ceres are not built with Cuda support. Right now it's silent and a bit confusing.

Reply:
Good idea. Will do.